src/components/functionselect/functionselect.js

```diff
@@ -14,25 +14,50 @@ angular.module('vlui')
 
         scope.func = {
           selected: undefined,
-          list: [undefined]
+          list: {
+            aboveFold: [],
+            belowFold: [] // could be empty
+          }
         };
 
-        function getFns(type) {
+        // timeUnits for T
+        var timeUnits = {
+          aboveFold: ["year", "quarter", "month", "date", "day", "hours", "yearmonthdate"],
```

Comment:
move minutes and seconds up here.

// TODO(later): use statistics to determine what's above fold and below fold (Waiting for CompassQL schema to provide this info)